### ISSUE TITLE
Module kamodo_ccmc.readers.OpenGGCM.readOpenGGCM not found 

### ISSUE THREAD

**Issue by couveflor** [nasa/Kamodo#104]
Hello,
when importing the module kamodo_ccmc.readers.openggcm_gm_tocdf I get an error message (see the text below) saying that the module kamodo_ccmc.readers.OpenGGCM.readOpenGGCM does not exist.
I checked the files in the corresponding repository and the mentioned module does not exist indeed.
Could you please check whether you get the same error?
Thanks,
Simona

```
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
Input In [22], in <cell line: 3>()
      1 from kamodo_ccmc import readers
      2 from  kamodo_ccmc.readers import OpenGGCM
----> 3 import kamodo_ccmc.readers.openggcm_gm_tocdf

File ~/.local/lib/python3.10/site-packages/kamodo_ccmc/readers/openggcm_gm_tocdf.py:13, in <module>
     10 from glob import glob
     11 #import kamodo_ccmc.readers.OpenGGCM.read_b_grids as rbg
     12 #import kamodo_ccmc.readers.OpenGGCM.readmagfile3d as rmhd
---> 13 import kamodo_ccmc.readers.OpenGGCM.readOpenGGCM as ropgm
     15 from os.path import sep,isfile,isdir,dirname,exists
     16 from os import remove

ModuleNotFoundError: No module named 'kamodo_ccmc.readers.OpenGGCM.readOpenGGCM'
```

**Comment by rebeccaringuette** [comment]
Hi couveflor,
Thanks for reporting the issue. We are aware of the problem. The error comes from a problem we have with the file conversion script for OpenGGCM global magnetosphere outputs, particularly that it won't run on other's machines (this is related to some Fortran version problems). The solution we have for it is for you to request the properly converted files from the CCMC (Lutz Rastaetter), download those files in the same directory as the original compressed binaries, and run it again. The block of code calling the file converter will no longer trigger as the converted files will be detected, and the rest should be smooth sailing. I will add a note on this to the top of the OpenGGCM_GM model reader doc string and an error message to the appropriate place so others get the same message. Thanks.
Rebecca

**Comment by darrendezeeuw** [comment]
Fixed in PR #109
